app/main.py

```diff
@@ -1 +1,41 @@
-#  write your code here
+class Animal:
+
+    def __init__(self, name: str, appetite: int, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        self.appetite = 0
+        return self.appetite
+
+
+class Cat(Animal):
+
+    def __init__(self, name: str, appetite=3, is_hungry=True):
+        super().__init__(name, appetite, is_hungry)
+
+    @staticmethod
+    def catch_mouse():
+        print("The hunt began!")
+
+
+class Dog(Animal):
+
+    def __init__(self, name: str, appetite=7, is_hungry=True):
+        super().__init__(name, appetite, is_hungry)
+
+    @staticmethod
+    def bring_slippers():
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list):
+    return sum([animal.feed() for animal in animals])
```

Comment:
```suggestion
    return sum(animal.feed() for animal in animals)
```
You could avoid creating an extra list

Reply:
Fixed, Thanks for the tip